HackIllinois/Models/APIModels/HIAPIEvent.swift

```diff
@@ -11,32 +11,37 @@
 //
 
 import Foundation
+import APIManager
 
-struct HIAPIEvent: Codable {
-    typealias Contained = HIAPIReturnDataContainer<HIAPIEvent>
+struct HIAPIEventsContainer: Decodable, APIReturnable {
+    let events: [HIAPIEvent]
+}
+
+struct HIAPIEvent: Codable, APIReturnable {
+    typealias Contained = HIAPIEventsContainer
```

Comment:
remove this

Reply:
drop APIReturnable if its not needed

Reply:
alright

Reply:
APIReturnable is necessary, as create returns an HIAPIEvent

Reply:
mmk